tests/karate/src/test/java/be/rollCall/createRollCall.feature

```diff
@@ -15,104 +15,194 @@ Feature: Create a Roll Call
-    Given string badRollCallReq  = read('classpath:data/rollCall/bad_roll_call_create_empty_data_but_same_messageId_as_valid_roll_call.json')
-    * call read('classpath:be/utils/simpleScenarios.feature@name=valid_lao')
-    * string rollCallData = read('classpath:data/rollCall/data/rollCallCreate/valid_roll_call_create_data.json')
-    * string rollCallCreate = converter.publishМessageFromData(rollCallData, rollCallCreateId, laoChannel)
-
-    When eval frontend.send(rollCallCreate)
-    * karate.log('Request for roll call sent')
-    * frontend_buffer.takeTimeout(timeout)
-
-    Then eval frontend.send(badRollCallReq)
-    * json roll_err = frontend_buffer.takeTimeout(timeout)
-    Then match roll_err contains deep {jsonrpc: '2.0', id: '#(rollCallCreateId)', error: {code: -4, description: '#string'}}
+    Given call read('classpath:be/utils/simpleScenarios.feature@name=valid_lao')
+    And def validCreateRollCall =
+      """
+        {
+          "object": "roll_call",
+          "action": "create",
+          "id": '#(getRollCallValidId)',
+          "name": "",
+          "creation": 1633098853,
+          "proposed_start": 1633099125,
+          "proposed_end": 1633099140,
+          "location": "EPFL",
+          "description": "Food is welcome!"
+        }
+      """
+    When frontend.publish(JSON.stringify(validCreateRollCall), laoChannel)
+    And json answer = frontend.getBackendResponseWithoutBroadcast()
+    Then match answer contains INVALID_MESSAGE_FIELD
+    And match frontend.receiveNoMoreResponses() == true
 
   # Setting up the lao correctly and sending a roll call create message that comes from
   # a non-organizer should result in an error message being sent by the backend.
   Scenario: Roll Call Creation with non-organizer as sender should return an error
-    Given string badRollCallReq = read('classpath:data/rollCall/bad_create_roll_call_not_organizer_sender.json')
-    And string badRollCallData = read('classpath:data/rollCall/data/rollCallCreate/valid_roll_call_create_data.json')
-    * string badRollCallCreate = converter.publishМessageFromData(badRollCallData, rollCallCreateId, laoChannel)
-
-    * call read('classpath:be/utils/simpleScenarios.feature@name=valid_lao')
-    When eval frontend.send(badRollCallReq)
-    * json roll_err = frontend_buffer.takeTimeout(timeout)
-    Then match roll_err contains deep {jsonrpc: '2.0', id: '#(rollCallCreateId)', error: {code: -4, description: '#string'}}
+    Given call read('classpath:be/utils/simpleScenarios.feature@name=valid_lao')
+    And def validCreateRollCall =
+      """
+        {
+          "object": "roll_call",
+          "action": "create",
+          "id": '#(getRollCallValidId)',
+          "name": "Roll Call ",
+          "creation": 1633098853,
+          "proposed_start": 1633099125,
+          "proposed_end": 1633099140,
+          "location": "EPFL",
+          "description": "Food is welcome!"
+        }
+      """
+    * frontend.setNonAttendeeAsSender()
```

Comment:
I tried giving it a better name now. But nonetheless with the test description it is quite descriptive I think, but I am maybe not objective enough since I worked on it.